ballerina/tests/error.bal

```diff
@@ -34,8 +34,8 @@ function TestAuthenticationError() {
     test:assertTrue(dbClient is sql:ApplicationError);
     error sqlerror = <error>dbClient;
     test:assertTrue(strings:includes(sqlerror.message(), "Error in SQL connector configuration: Failed to " +
-                "initialize pool: Wrong user name or password [28000-206] Caused by :Wrong user name or password " +
-                "[28000-206]"), sqlerror.message());
+                "initialize pool: Wrong user name or password [28000-212] Caused by :Wrong user name or password " +
+                "[28000-212]"), sqlerror.message());
```

Comment:
```suggestion
                "initialize pool: Wrong user name or password "), sqlerror.message());
```
Lets remove version number from verifying string

Reply:
Fixed in [1fbc2da](https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/451/commits/1fbc2da299afa65315fae62eba96da1607c6a6db)